site/source/tutorials/006_deploying_to_production/index.md

```diff
@@ -0,0 +1,143 @@
+
+The next two arguments that are listed when the `dojo build help` command is run are the `--element` and `--elementPrefix`. These two fields are used when exporting Dojo 2 widgets as [web components](https://en.wikipedia.org/wiki/Web_Components). They are not used when building an application.
+
+The `--debug` argument will cause the builder to generate an additional file - `profile.json`. This file contains a great deal of information about the build itself, including what modules were included and how long they took to build. A profile be inspected using WebPack's [analyze tool](https://webpack.github.io/analyse/).
+
+The final argument is `--disableLazyWidgetDetection`. A widget can be registered with the [widget registry](https://github.com/dojo/widget-core#widget-registry) to load only when required. This [lazy loading](https://en.wikipedia.org/wiki/Lazy_loading) behavior can reduce the initial load time of the application and is often useful when large, sophisticated widgets are not always required. The build system does this by creating a separate bundle for each of these lazily loaded widgets. Lazy loading can, however, cause unnecessary delays if the widgets are often needed. In those situations, adding the `--disableLazyWidgetDetection` argument will cause the the build system to include lazily loaded widgets into the main bundle. The widgets will then be immediately available for use by the application.
+
+The command line arguments give control over how the application is built and prepared for deployment. However, providing these arguments every time a build is required can be error prone. In the next section, we will learn how to create a persistent build configuration by using `.dojorc` files.
+
+{% section %}
+
+## Configuration files
+
+{% task 'Use a .dojorc file to store configuration settings.' %}
+
+The command line arguments that the `dojo build` tool accepts are very useful when developing an application and preparing it to be deployed to production. However, since command line arguments can't be committed to source control, they are not the best way to ensure that production deployment builds are executed consistently. To store configuration settings, Dojo 2 applications can store their settings in a file called `.dojorc`.
```

Comment:
store their settings in a file called `.dojorc`. -> store their settings in `.dojorc`.